tests/acceptance/clinical_browser.js

```diff
@@ -118,13 +118,15 @@ function clinVarSearchByAccession(driver){
     driver.findElement(By.name("accessionId")).clear();
     driver.findElement(By.name("accessionId")).sendKeys("RCV000074666");
     driver.findElement(By.xpath("//div[contains(@id,'ClinvarWidgetPanel')]//span[text()='Submit']")).click();
+    driver.sleep(5000);
```

Comment:
Isn't there absolutely no other way to do this than a sleep? Well, I see these are just in tests, but still.